### DISCUSSION TITLE
LED gaat bijna niet aan

### DISCUSSION THREAD

**Discussion by cam-emmauscollege** [emmauscollege/informatica2021#5] [Arduino hulp] [answered]
Als ik `digitalWrite(4, HIGH)` uitvoer, gaat de LED maar een heel klein beetje branden. Heeft iemand een idee wat ik fout doe?

**Comment by cam-emmauscollege** [comment]
Heb je wel de goede weerstand gebruikt?

**Reply by vangeest** [reply]
Heb je ergens kortsluiting?

**Reply by cam-emmauscollege** [reply]
ja, regelmatig in mijn bovenkamer, maar dat lijkt nu het probleem niet te zijn. 🤔

**Reply by cam-emmauscollege** [reply]
> ja, regelmatig in mijn bovenkamer, maar dat lijkt nu het probleem niet te zijn. 🤔

Test auote

**Comment by cam-emmauscollege** [comment, marked as answer]
Dit kan door 3 oorzaken komen:
- Je hebt alles goed gedaan, maar deze ene LED is minder fel. Probeer eens een andere LED en kijk wat het effect is.
- Je hebt een weerstand gebruikt met een te hoge waarde. Gebruik voor het gemak bij iedere LED die van 220Ω (rood-rood-bruin)
- Je bent vergeten om bij `setup` `pinMode(4 OUTPUT)` aan te geven.

**Comment by vangeest** [comment]
Test geslaagd?

Sander van Geest
gee@emmauscollege.nl
0613099989

Op do 24 dec. 2020 20:23 schreef cam-emmauscollege <notifications@github.com
